sensor_msgs/RegionOfInterest roi
---
int8 error
 
